### PR TITLE
Add review files

### PR DESCRIPTION
**Pull request by lampapos**

### CODE REVIEW

File: game_of_life_review.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    field_size = [int(i) for i in f.readline().split(" ")
+                  ]  # 1 argument = y, 2 argument = x
+    first_gen = [[j for j in i.rstrip("\n")] for i in f.readlines()]
+
+
+def new_cell_state(field_state: list, cell_x: int, cell_y: int) -> str:
+    # Coordinates which are NOT placed on borders
+    if (cell_x > 0 and cell_x < (field_size[1]-1)) and (cell_y > 0 and cell_y < (field_size[0]-1)):
+        cells_neighbours = [
+            field_state[cell_x-1][cell_y-1],
```
Comment (lampapos) [inline game_of_life_review.py:12]: Wow! A lot of corner cases :) I would recommend making a method like `getActualCoords(x, y, sizeX, sizeY)` that is converting coordinates outside of the field to actual field coordinates.

For example, you have a board 3x3 (0:0 = top left corner):

```
getActualCoords(0, 0, 3, 3) => 0;0
getActualCoords(1, 2, 3, 3) => 1;2

getActualCoords(1, 0, 3, 3) => 2;0
getActualCoords(3, 0, 3, 3) => 0;0
```

With this method you could minimize your 9 branches to 1

File: game_of_life_test_review.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import pytest
+import game_of_life
+import test
+
+
+def test_newCellStateTrue():
+    assert game_of_life.new_cell_state(game_of_life.first_gen, 1, 3) == "."
```
Comment (lampapos) [inline game_of_life_test_review.py:7]: Remind, plz, what we discussed on the lection about unit test - a test should give answers, don't raise them

My questions to this test:
1. what is first_gen? Suggestion: put small test field directly in the test
2. what is `1, 3`? Suggestion: in Python you have named arguments - `game_of_life.new_cell_state(game_of_life.first_gen, cell_x=1, cell_y=3)`
3. what is `.`? Actually, it's an input/output detail - it might be better to use boolean value here and convert it into character in some `output` method

But overall it's awesome that you have this test!

File: game_of_life_test_review.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+
+def test_newCellStateFalse():
+    assert game_of_life.new_cell_state(game_of_life.first_gen, 1, 3) != "x"
```
Comment (lampapos) [inline game_of_life_test_review.py:11]: You have so many corner cases (9 if branches) and only 2 tests for them - feels like not enough

File: game_of_life_test_review.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+
+@pytest.mark.parametrize("listik, x, y, answer", [(game_of_life.first_gen, 1, 3, "."), (game_of_life.first_gen, 0, 3, ".")])
```
Comment (lampapos) [inline game_of_life_test_review.py:14]: Nice idea! But why not include the previous two tests here?

Comment (AsyaAndKo) [reply game_of_life_test_review.py:14]: Those tests are more for training, to be honest. So I've tried to write simple tests and after that I was interested in how to parametrize them (lot's of learning parts here)

File: game_of_life_test_review.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    assert game_of_life.new_cell_state(listik, x, y) == answer
+
+
+def test_gameOfLifeOutput():
+    test_text = open("test.txt", "r")
+
+    output_list = game_of_life.game_of_life(
+        game_of_life.first_gen, game_of_life.gen_num)
+    test_text_list = [[j for j in i.rstrip("\n")]
+                      for i in test_text.readlines()]
```
Comment (lampapos) [inline game_of_life_test_review.py:25]: 🤯
